src/binaryapi/ActiveSymbols.js

```diff
@@ -14,16 +14,18 @@ export default class ActiveSymbols {
         this._tradingTimes = tradingTimes;
     }
 
-    @action.bound async retrieveActiveSymbols() {
-        if (this.isRetrievingSymbols) {
+    @action.bound async retrieveActiveSymbols(retrieve_new_active_symbols = false) {
```

Comment:
@kellybinary for the sake of consistency.
```suggestion
    @action.bound async retrieveActiveSymbols(retrieveNewActiveSymbols = false) {
```